apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-description/edit.js

```diff
@@ -45,33 +47,72 @@ class SiteDescriptionEdit extends Component {
 		}
 
 		if ( ! prevProps.shouldUpdateSiteOption && shouldUpdateSiteOption ) {
-			apiFetch( { path: '/wp/v2/settings', method: 'POST', data: { description } } )
-				.then( () => this.updateInitialDescription() )
-				.catch( ( { message } ) => {
-					noticeOperations.createErrorNotice( message );
-					this.revertDescription();
-				} );
+			this.saveSiteDescription( description );
 		}
 	}
 
-	revertDescription = () => this.setState( { description: this.state.initialDescription } );
+	onSave = () => {
+		const { description, initialDescription } = this.state;
+		const descriptionUnchanged = description && description.trim() === initialDescription.trim();
```

Comment:
could potentially pull this out to a checkDescriptionUnchanged() method as assigning this in two places

Reply:
Is this worth checking? initialDescription is only fetched once on mount, and may become stale/out of date if we leave the editor open long enough. 

We also run into a similar situation if someone updates values at the same time from an editor / options.